authentication/managing-oauth-access-tokens.adoc

```diff
@@ -0,0 +1,17 @@
+[id="managing-oauth-access-tokens"]
+= Managing user OAuth access tokens
+include::modules/common-attributes.adoc[]
+:context: managing-oauth-access-tokens
+
+toc::[]
+
+Users can manage their own OAuth access tokens.
```

Comment:
Would a bit of context about actions that users can't do themselves or what an admin must have done before a user can do anything be helpful here?

Reply:
Not that I can think of. Pretty much users can request their own tokens, and then now they can actually see and delete them if they wanted to. Admins can just manage other users' tokens, but that's using a different resource. Nothing that admins should need to do in advance. But I'll see if I can give any more detail here, it is pretty minimal.